doc/gallery-src/framework/run_blockVolumeFractions.py

```diff
@@ -15,7 +15,7 @@
 # -*- coding: utf-8 -*-
 """
 Computing Component Volume Fractions on a Block with Automatic Thermal Expansion
-================================================================================
+================================================================================.
```

Comment:
This one is a little strange at first sight

Reply:
Fixed. Sorry for the trouble.